regexes/device/mobiles.yml

```diff
@@ -7057,7 +7181,7 @@ Spice:
 
 # Sharp
 Sharp:
-  regex: 'SHARP|SBM|SH-?[0-9]+[a-z]?[);/ ]|AQUOS|506SH|SHL22 '
+  regex: 'SHARP|SBM|SH-?[0-9]+[a-z]?[);/ ]|AQUOS|506SH|SHL22|FS8010 '
```

Comment:
Actually I'm nor sure if the ` ` in the end of `SHL22` was there on purpose. But as there are no tests that fail I guess it wasn't. Nevertheless I think it might be better to match exact model fragments like `SHL22` or `FS8010` with a `[);/ ]` in the end, so it doesn't match anything with additional chars